nh_eobs_mental_health/models/nh_clinical_patient_observation_ews.py

```diff
@@ -109,6 +113,12 @@ def schedule_clinical_review_notification(self, activity_id):
         activity_model = self.env['nh.activity']
         activity = activity_model.browse(activity_id)
         still_valid = self.is_refusal_in_effect(activity_id, mode='child')
+
```

Comment:
The logging? I did it for debugging but thought it might be useful. What's our policy on logging? Should we be doing it everywhere at debug level just in case we need it?